DEVELOPING_A_RUNTIME.md

```diff
@@ -1,18 +1,18 @@
-# Builders Developer Reference
```

Comment:
Since you are renaming the file, you'll probably need to update the docs that link here

Reply:
Yep!

Reply:
Could make a symlink 😄

Reply:
I almost think we should move this document to the wiki

https://github.com/zeit/now/wiki